python/lsst/pipe/tasks/getRegionTimeFromVisit.py

```diff
@@ -33,13 +33,10 @@ class GetRegionTimeFromVisitConnections(pipeBase.PipelineTaskConnections,
                                         defaultTemplates={"coaddName": "goodSeeing", "fakesType": ""}):
```

Comment:
Possibly add a comment that these templates are kept in case `dummy_visit` is a dataset type that needs them?